frontend/src/Users/Users.tsx

```diff
@@ -1,5 +1,173 @@
+import DialogTitle from '@mui/material/DialogTitle';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import { useTranslation } from "react-i18next";
+
+interface EditUserProps {
+  open: boolean;
+  name: string;
+  closeDialog: () => void;
+}
+
+function EditUserDialog({ open, name, closeDialog }: EditUserProps) {
+  const { t } = useTranslation();
+  const [role, setRole] = React.useState('guest');
+  return (
+    <Dialog open={open} onClose={closeDialog}>
+      <DialogTitle>{t('users.dialog.title')}</DialogTitle>
+      <DialogContent>
+        <TextField
+          autoFocus
+          margin="dense"
+          id="username"
+          label="Username"
+          type="text"
+          fullWidth
+          variant="standard"
+          placeholder="Username"
+        />
+        <InputLabel id="role-label" sx={{display: 'inline-block', marginRight: '10px'}}>Role</InputLabel>
+        <Select
+          labelId="role-label"
+          id="demo-simple-select"
+          value={role}
+          label={t('users.role')}
+          onChange={(event) => {setRole(event.target.value)}}
+        >
+          <MenuItem value="editor">{t('users.dialog.editor')}</MenuItem>
+          <MenuItem value="intern">{t('users.dialog.intern')}</MenuItem>
+          <MenuItem value="guest">{t('users.dialog.guest')}</MenuItem>
+        </Select>
+        <FormGroup>
+        <FormControlLabel
+          control={<Checkbox name="isBlocked" />}
+          label={t('users.dialog.block-access')}
+        />
+        </FormGroup>
+        <TextField
+          autoFocus
+          margin="dense"
+          id="fullName"
+          label={t('users.full-name')}
+          type="text"
+          fullWidth
+          variant="standard"
+          value={name}
+          placeholder={t('users.full-name')!!}
+        />
+        <TextField
+          autoFocus
+          margin="dense"
+          id="password"
+          label={t('users.dialog.new-password')}
+          type="password"
+          fullWidth
+          variant="standard"
+          placeholder={t('users.dialog.new-password')!!}
+        />
+        <TextField
+          autoFocus
+          margin="dense"
+          id="comment"
+          label={t('users.dialog.comment')}
+          type="text"
+          fullWidth
+          variant="standard"
+          placeholder={t('users.dialog.comment')!!}
+          multiline
+          minRows={2}
+        />
+      </DialogContent>
+      <DialogActions>
+        <Button onClick={closeDialog}>{t('users.dialog.cancel')}</Button>
+        <Button onClick={closeDialog}>{t('users.dialog.save')}</Button>
+      </DialogActions>
+    </Dialog>
+  );
+}
+
+
+function createData(
```

Comment:
Isn't it easier to write just `{ role, fullName }` without this function?..

Reply:
Mmm it was just a template from internet, so perhaps you are right :) I'll check